kernel/init.c

```diff
@@ -190,39 +235,49 @@ static void bg_thread_main(void *unused1, void *unused2, void *unused3)
 	z_mem_manage_init();
 #endif /* CONFIG_MMU */
 	z_sys_post_kernel = true;
+	set_sof_status(0x33330002);
+
+	/* Save and display my_int_mask_3_cnt from _xtensa_handle_one_int3()
+	 *
+	 * Verify how many times the interrupt handler was executed so far
+	 */
+	init_bg_thread_main_mask_3_cnt = my_int_mask_3_cnt;
```

Comment:
More debug code?

Reply:
Yes, more debug code, part of [do not merge] commit https://github.com/zephyrproject-rtos/zephyr/pull/35467/commits/ea08a79fb2974dad505e9b7274b95bbd5ca2cd62